doc/releases/migration-guide-4.3.rst

```diff
@@ -75,6 +75,12 @@ Display
   This has now been fixed. Boards and applications that were tested or developed based on the
   previous sample may be affected by this change (see :github:`79996` for more information).
 
+PTP Clock
+*********
+
+* Changed the function of :c:func:`ptp_clock_rate_adjust` API for general ratio adjustment
+  based on nominal frequency, but not for only gPTP neighbor ratio adjustment.
```

Comment:
Heya, this does not sound too clear to me, what's the action that a user should take? How an application should be changed to adapt to the new behavior?

Reply:
@fabiobaltieri 

Let me clarify it.

1. The API implementation was defined by device drivers. Previously, it's used by gPTP stack to adjust rate ratio based on its current frequency.
2. In v4.2, I added PI servo in both gPTP and PTP. Changed to use it to adjust rate ratio based on nominal frequency which is more general, also with ENET PTP driver updating the implementation.

Now. I am converting other device drivers implementation, and update the API doc.
This PTP clock API is used by zephyr net gPTP and PTP stacks, no action need to take.
Not sure if there are other users use it for other purpose.

Do you need me to add more detail in release note?
Thanks.

Reply:
> Now. I am converting other device drivers implementation, and update the API doc.
> This PTP clock API is used by zephyr net gPTP and PTP stacks, no action need to take.

Ok then if I understand correctly there's no action for the user to take at application level, right? Though there may be a change required for an external driver implementing this API?

Reply:
Ok that's cool, can you reword this on the line of "drivers implementing `ptp_clock_rate_adjust` should be adjusted to account for the new behavior"?

Basically what I look for the migration guide is some clear wording for user to figure if:
1. am I affected by the change
2. what do I need to do

I'll add some notes to the file about it, sorry I realize there's probably plenty of counter-example in the file already, try to catch what I can

Reply:
Thanks @fabiobaltieri . I should have had clear description in notes.
I updated the migration note. Could you help to check if it can address your comments.

Thanks a lot.